docs/topic/graph-matching/bigraph-match.md

```diff
@@ -1,32 +1,40 @@
 author: accelsao
 
-# 二分图最大匹配
+# Maximum Bipartite Matching
 
-为了描述方便将两个集合分成左和右两个部分，所有匹配边都是横跨左右两个集合，可以假想成男女配对。
+A **matching** of a graph is a sub set of edges such that no two edges share a vertex. A **maximum matching** of a graph is a matching with the maximum number of edges(matching as many vertices as possible).  
 
-假设图有 $n$ 个顶点， $m$ 条边。
+For the convenience of description, we partition the graph into two disjoint sets - left and right. All matching edges connects to both left and right sets. One common example of bipartite graph matching is matching a group of men and women.
 
-## 增广路算法 Augmenting Path Algorithm
+Assume the graph has $n$ vertices and $m$ edges.
 
-因为增广路长度为奇数，路径起始点非左即右，所以我们先考虑从左边的未匹配点找增广路。
-注意到因为交错路的关系，增广路上的第奇数条边都是非匹配边，第偶数条边都是匹配边，于是左到右都是非匹配边，右到左都是匹配边。
-于是我们给二分图 **定向** ，问题转换成，有向图中从给定起点找一条简单路径走到某个未匹配点，此问题等价给定起始点 $s$ 能否走到终点 $t$ 。
-那么只要从起始点开始 DFS 遍历直到找到某个未匹配点， $O(m)$ 。
-未找到增广路时，我们拓展的路也称为 **交错树** 。
+## Augmenting Path Algorithm
 
-因为要枚举 $n$ 个点，总复杂度为 $O(nm)$ 。
+Because the length of augmenting path is odd, and the starting vertex of the path is either on the left or right, we first consider finding the augmenting path from the unmatched vertices on the left.
 
-### 代码
+Please note that because of the [alternating path](./augment.md), the odd-numbered edges on the augmenting path are all unmatched edges, and the even-numbered edges are all matched edges. So left-to-right are unmatched edges, and right-to-left are matched edges.
```

Comment:
```suggestion
Please note that because of the [alternating path](./augment.md), the odd-numbered edges on the augmenting path are all unmatched edges, and the even-numbered edges are all matched edges. So left-to-right edges are all unmatched, and right-to-left edges are all matched.
```